apps/console/src/public/deployment.config.json

```diff
@@ -469,6 +469,24 @@
                     ]
                 }
             },
+            "smsProviders": {
+                "disabledFeatures": [],
+                "enabled": true,
+                "scopes": {
+                    "update": [
+                        "internal_config_mgt_update"
+                    ],
+                    "delete": [
+                        "internal_config_mgt_delete"
+                    ],
+                    "create": [
+                        "internal_config_mgt_add"
+                    ],
+                    "read": [
+                        "internal_config_mgt_view"
+                    ]
+                }
+            },
```

Comment:
Hope you added these properties to the framework as well

Reply:
These are already exists in framework